compiler/resolver.stanza

```diff
@@ -815,10 +911,20 @@ defn SymbolTable (package-name:Symbol, base:NameMap|Tuple<Export>) :
     do(add-entry{_, false}, es)
 
   ;Importing a sequence of entries
-  defn import (es:Seqable<VEntry>, prefix:Tuple<IPrefix>) :
-    val ptable = PrefixTable(prefix)
+  defn import (es:Seqable<VEntry>, ptable:PrefixTable, pred:(VEntry -> True|False)) :
+    for e in es do :
+      if pred(e) : add-entry(e, ptable[name(e)])
+
+  ;Forward protected definitions by copying and updating their owner
+  defn forward-protected (es:Seqable<VEntry>) :
```

Comment:
Let's have `protected` definitions also gain a prefix during this phase.

Reply:
Let's also have `protected` definitions satisfy the `only` filter.

Reasoning: When forwarding a package, I as a user use the `only` filter for the purpose of explicitly curating the set of names that my API exposes.